bucket/networkmanager.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0",
+    "suggest": {
+        "vcredist": "dotnet-sdk"
+    },
+    "##": "only desktop runtime from dotnet is required",
```

Comment:
```suggestion
```

Reply:
This is not true.

![image](https://user-images.githubusercontent.com/13260377/105349774-7c405d80-5bea-11eb-9815-cb23bf56f2d1.png)

Reply:
This comment was meant for dotnet-sdk only. Honestly I am not even sure whit brought WebView2 to ma PC.